.travis.yml

```diff
@@ -13,4 +18,6 @@ before_script:
         else
             PYTEST_ADDOPTS="-m 'not optional'"
         fi
-script: make PYTEST_ADDOPTS="${PYTEST_ADDOPTS}" test-docker
+script:
+    - docker version
```

Comment:
It does show, but collapsed and not highlighted in green like now:
**The command "docker version" exited with 0.**

I guess it could be sometimes useful to see what docker version has been used for testing, so I'd rather keep it.